install/gce-installation-guide.md

```diff
@@ -0,0 +1,119 @@
+
+## Creating an Image with Nested Virtualization Enabled
+
+VM images on GCE are grouped into families under projects. Officially supported images are automatically discoverable with `gcloud compute images list`. That command will produce a list similar to the following (likely with different image names):
+
+    $ gcloud compute images list
+    NAME                                                  PROJECT            FAMILY                            DEPRECATED  STATUS
+    centos-7-v20180523                                    centos-cloud       centos-7                                      READY
+    coreos-stable-1745-5-0-v20180531                      coreos-cloud       coreos-stable                                 READY
+    cos-beta-67-10575-45-0                                cos-cloud          cos-beta                                      READY
+    cos-stable-66-10452-89-0                              cos-cloud          cos-stable                                    READY
+    debian-9-stretch-v20180510                            debian-cloud       debian-9                                      READY
+    rhel-7-v20180522                                      rhel-cloud         rhel-7                                        READY
+    sles-11-sp4-v20180523                                 suse-cloud         sles-11                                       READY
+    ubuntu-1604-xenial-v20180522                          ubuntu-os-cloud    ubuntu-1604-lts                               READY
+    ubuntu-1804-bionic-v20180522                          ubuntu-os-cloud    ubuntu-1804-lts                               READY
+
+Each distribution has its own project, and each project may host images for multiple versions of the distribution, typically grouped into families. We recommend selecting images by project and family rather than by name as it ensures any scripts or other automation will always be working with a non-deprecated image including security updates, updates to GCE-specific scripts, etc.
+
+### Creating the Image
+
+The following (substituing your preferred disribution project and image family) will produce an image with nested virtualization enabled in your currently active GCE project:
+
+```bash
+SOURCE_IMAGE_PROJECT=ubuntu-os-cloud
+SOURCE_IMAGE_FAMILY=ubuntu-1804-lts
+IMAGE_NAME=${SOURCE_IMAGE_FAMILY}-nested
+
+gcloud compute images create \
+    --source-image-project $SOURCE_IMAGE_PROJECT \
+    --source-image-family $SOURCE_IMAGE_FAMILY \
+    --licenses=https://www.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx \
+    $IMAGE_NAME
+```
+
+`gcloud` should report that the image was created. You can verify that the image has the nested virtualization license with `gcloud compute images show $IMAGE_NAME`. This should produce output like the following (some fields have been removed for clarity and to redact personal info):
```

Comment:
To stay active: 

`gcloud` reports that the image was created. Verify that the image has the nested virtualization license with `gcloud compute images show $IMAGE_NAME`. This produces output like the following (some fields have been removed for clarity and to redact personal info):

Reply:
Done, although I've added a conditional clause to recognize the possibility of failure.